File: Platron/Tests/BaseTest.cs
using System;
using NUnit.Framework;
using Platron.Core;
using Platron.Core.Service;

namespace Platron.Tests
{
    public class BaseTest : IDisposable
    {
        public BaseTest()
        {
            //RunSettings.ConfigurationFiles();
            Print("Constructor");
            ServiceDriver.initilize();
        }

        public void Print(string m)
        {
            TestContext.Error.WriteLine(m);
        }
        [TearDown]
        public void Close()
        {
            Print("TearDown");
        }

        public void Dispose()
        {
            Print("Dispose");
            ServiceDriver.Quit();
        }
    }
}
    

File: Platron/src/test/java/BaseTest.java
import org.junit.After;
import org.junit.Before;
import org.junit.BeforeClass;

public class BaseTest {
//    @BeforeClass
//    public static void onceExecutedBeforeAll() {
//        System.out.println("@BeforeClass: onceExecutedBeforeAll");
//    }
    @Before
    public void Before() throws Exception {
            ServiceDriver.initilize();
            System.out.println("Before");
    }

    @After
    public void After()
    {
        ServiceDriver.Quit();
        System.out.println("After");
    }
}
